src/kibana/templates/deployment.yaml

```diff
@@ -51,6 +51,9 @@ spec:
             - name: {{ template "kibana.name" . }}
               mountPath: "/tmp/plugins"
               subPath: plugins
+            - name: {{ template "kibana.name" . }}
+              mountPath: "/usr/share/kibana/config/kibana.yml"
```

Comment:
Please see my comments on the upstream PR WRT exposing the entire config at values.yaml